### ISSUE TITLE
Hello, I opened a server but I can't remove the edit blocks.

### ISSUE THREAD

**Issue by xaviertuto1122** [Divine-Journey-2/Divine-Journey-2#1086]
### Mod Name

ftb utilities

### Reasons why this should be added

I opened a hosting and I want to play with some friends, I disabled editing in my chunck for everyone but they can still edit, please help me, I'm looking for a way to solve it but I can't find a way, sorry for my English, I'm using Google Translate

### Your Divine Journey 2 Discord Username

246731873890467840

**Comment by Preta-Crowz** [comment]
at default, op permission includes bypass ftbu chunk protection, make sure your friend don't have that

**Comment by xaviertuto1122** [comment]
Yes, but I take off the top or the top is removed and we can still break blocks within the server

**Comment by xaviertuto1122** [comment]
and even though I don't belong to the team, it doesn't prevent me from breaking blocks or opening chests.
